package.json

```diff
@@ -22,24 +22,23 @@
     "@types/jest": "^26.0.22",
     "@types/lodash.merge": "^4.6.6",
     "@types/node": "^12.12.7",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/experimental-utils": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
-    "@typescript-eslint/typescript-estree": "^4.21.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.0",
+    "@typescript-eslint/parser": "^5.42.0",
+    "@typescript-eslint/typescript-estree": "^5.42.0",
     "codecov": "^3.6.1",
-    "eslint": "^6.6.0",
+    "eslint": "^8.26.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.7.0",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^23.0.4",
     "jest": "^26.6.3",
     "prettier": "^2.2.1",
     "rimraf": "^3.0.0",
     "ts-jest": "^26.5.5",
-    "typescript": "^4.2.4"
+    "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "eslint": ">5.0.0"
+    "@typescript-eslint/utils": ">4.0.0"
```

Comment:
imho, this should appear also in devDependencies. The package users (library consumers) don't care, but might be handy for local development ;)